static/js/profile.js

```diff
@@ -0,0 +1,539 @@
+/* Code by Milos Kozak
+
+TODO:
+	- treatment drawer must be updated to enter glycemic index
+	- update calculation to use new style of values
+	- how to handle units? store native or convert to mg/dL?
+
+*/
+	var profile = {};
+	var apisecret = "", storeapisecret = false;
+	
+	var defaultprofile = {
+			//General values
+			"simple": true,
+			"calculator": false,
+			"dia":3,
+			
+			// Simple style values, "from" are in minutes from midnight
+			"carbratio":24,
+			"carbs_hr":30,
+			"sens":17,
+			
+			"validfrom": new Date(),
+			
+			//Bolus calculator style values, "from" are in minutes from midnight
+			"ic": [{"from":0,"val":24},
+					{"from":0,"val":0},
+					{"from":0,"val":0},
```

Comment:
Would be way cleaner if the number of entries was controlled programmatically and this didn't assume a set number of values.